.gitignore

```diff
@@ -5,5 +5,4 @@
 *.tgz
 *.lock
 **/*.lock
-.pytest_cache
```

Comment:
Can you please correct the ignore as in https://github.com/EcovadisCode/charts/pull/46/commits/a1b1ae4fa98379a8b1bff9dd00f6f9afc99f744b ?